_projects/2023/100483064/100483064.Rmd

```diff
@@ -0,0 +1,440 @@
+---
+title: "Graph: La evolución del despligue militar de EE.UU."
+description:
+  "This project will explain the process of replicating the graphic La evolución del despliegue militar de EE.UU. created by El Orden Mundial."
```

Comment:
No need to repeat the title, please explain a bit what this graph is about to add a bit of context.